README.md

```diff
@@ -16,21 +19,27 @@ toolchain. Most developers will interact with Flutter via the [Flutter
 Framework](https://github.com/flutter/flutter), which provides a modern,
 reactive framework, and a rich set of platform, layout and foundation widgets.
 
-If you want to run/contribute to Flutter Web engine, more tooling can be
+To Run/contribute into Flutter Web engine, more tooling can be
```

Comment:
uber-nit: Run -> run ;)

Reply:
"To Run/contribute into Flutter Web engine" -> To run/contribute into Flutter Web engine?

Reply:
anyway just uber nit